apply/templates/apply/fund_start_page.html

```diff
@@ -85,7 +85,7 @@ <h2 class="govuk-heading-m">{% trans %}Get help{% endtrans %} </h2>
                 </p>
                 {% endif %}
                 <p class="govuk-body">
-                    {% trans %}Do not send any applications or attachments from the service desk portal. We'll only accept applications through this service.{% endtrans %}
+                    {% trans %}Do not send any applications or attachments through the support desk. We'll only accept applications through this service.{% endtrans %}
```

Comment:
All of these will change the translation "key" which will mean these snippets no longer load in Welsh, is that something we're OK with in the meantime or do we want to run the translate script and make sure that stays up to date?

(If these don't actually have Welsh translations at the moment then that would prove it doesn't matter?)

Reply:
Fine with me - can we make sure following up on that gets put in as a ticket on a backlog that will be played? Happy to do that for you if you helpful